fish_passage_fieldwork.Rmd

```diff
@@ -0,0 +1,37 @@
+# Fish Passage Field Work Guide
+
+## Field Work Prep
+
+### Apps and Programs Used
+
+1. [Timestamp Camera](http://www.timestampcamera.com/) is used to take photos (landscape only) in the field. Time and utm settings can be found in Zotero with the citation key `irvine2021FishPassage`.
```

Comment:
this is the reference word doc in zotero/fisheries/fish passage/methods/  .  1. its a reference key which is a bizarre mateo thing to put in the paragraph, zotero is out of scope (and unaccessible to most) for the document (for now), and the info in the doc that is good needs to transfer in to this document.